pilot/pkg/networking/core/v1alpha3/peer_authentication_simulation_test.go

```diff
@@ -0,0 +1,350 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha3_test
+
+import (
+	"testing"
+
+	"istio.io/istio/pilot/pkg/model"
+	"istio.io/istio/pilot/pkg/simulation"
+	"istio.io/istio/pilot/pkg/xds"
+)
+
+func TestPeerAuthenticationPassthrough(t *testing.T) {
```

Comment:
there is a very similar test in https://github.com/istio/istio/pull/28507, do you want to rebase on that?

Reply:
I moved the other test also to this file, I think they're largely testing different configuration combination (service or sidecar), there is a small overlap here but I prefer to keep both of them.